### ISSUE TITLE
Feature request: seconds support for scheduling

### ISSUE THREAD

**Issue by byung806** [swiftbar/SwiftBar#408]
For many timer/countdown apps, it is necessary to update the menu bar text every second. Unfortunately, cron syntax only supports down to minutes (due to only checking for cron tasks every minute on Linux).

However, this app is able to update every second. I am suggesting an expansion to the scheduling functionality for seconds:
Existing:
(0-59) (0-23) (1-31) (1-12) (0-6)  (minutes, hours, days, month, weekday)
Suggested:
(0-59) (0-59) (0-23) (1-31) (1-12) (0-6) (seconds, minutes, hours, days, month, weekday)

**Comment by melonamin** [comment]
Hey @byung806, you know that you can configure schedule using special syntax in the [file name](https://github.com/swiftbar/SwiftBar#plugin-naming)? There is support for seconds and even milliseconds. 

I'm using a small thirdparty library for cron-syntax parsing and I would rather not touch it.

**Comment by byung806** [comment]
Yes, I'm aware of using 1s or 1ms to update the item every second/ms, but I haven't figured out a way to only update when needed.
For context, I'm making a script that tracks how much time is left in my period at school (counts down to the next bell). It would need to update every second while school is going on, but when there's no school, for example on the weekends, I don't want the app to update at all.
Is this possible using the name special syntax? Or a combination of the name syntax with the scheduling cron syntax?

**Comment by melonamin** [comment]
Ah, I see. No, it is not possible with the current feature set. 

Let me think about it.

**Comment by byung806** [comment]
Please let me know if this becomes possible. Thank you for your efforts!

**Comment by mig8447** [comment]
I'd also like to get support for seconds, maybe just add another digit to the cron syntax and support up to 6 groups (in which case the first would be the seconds)?
